packages/runner-ct/src/app/app.tsx

```diff
@@ -86,18 +90,39 @@ const App: React.FC<AppProps> = observer(
     return (
       <>
         <main className="app-ct">
-          <div className={cs('specs-list-container', { 'specs-list-container__open': isSpecsListOpen })}>
-            <nav>
-              <a onClick={() => setIsSpecsListOpen(!isSpecsListOpen)} id="menu-toggle"
-                className="menu-toggle" aria-label="Open the menu">
-                <i className="fa fa-bars" aria-hidden="true"/>
-              </a>
-            </nav>
-            <SpecList
-              specs={state.specs}
-              selectedSpecs={state.spec ? [state.spec.absolute] : []}
-              onSelectSpec={(spec) => state.setSingleSpec(spec)}
-            />
+          <div
+            className="specs-list-drawer"
+            style={{
+              transform: isSpecsListOpen ? `translateX(0)` : `translateX(-${drawerWidth - 20}px)`,
+            }}
+          >
+            <ResizableBox
+              disabled={!isSpecsListOpen}
+              width={drawerWidth}
+              onIsResizingChange={setIsResizing}
+              onWidthChange={setDrawerWidth}
+              className="specs-list-container"
+              data-cy="specs-list-resize-box"
+              minWidth={200}
+              maxWidth={windowSize.width / 100 * 80} // 80vw
```

Comment:
If you open the specs-list at maximum, then reduce the size of the window, the spec list will not size down.
It feels like it should. But this can be taken care of in a future PR.

Reply:
I think that it is desired behavior. I am not sure one more resize observer is really required here..